lib/plugins/config/hapi-pino.js

```diff
@@ -1,23 +0,0 @@
-/*
-Copyright 2019 Expedia Group, Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-const env = (process.env.NODE_ENV || 'development').toLocaleLowerCase()
-
-const options = {
-  enabled: env !== 'test'
```

Comment:
There is no `enabled` option for `hapi-pino`. I'm guessing this was meant as a Steerage option for the plugin registration?